app/components/views/GetStartedPage/SetupWallet/ProcessUnmanagedTickets/ProcessUnmanagedTickets.jsx

```diff
@@ -1,9 +1,9 @@
 import { useState, useEffect } from "react";
 import { Tooltip } from "pi-ui";
 import { Subtitle } from "shared";
-import { GoBackMsg } from "../messages";
+import { GoBackMsg } from "../../messages";
 import { FormattedMessage as T } from "react-intl";
-import GetStartedStyles from "../GetStarted.module.css";
+import GetStartedStyles from "../../GetStarted.module.css";
```

Comment:
please add a TODO here to change this import and move all styles to component's css module.